crates/forge/tests/integration/store_load.rs

```diff
@@ -0,0 +1,597 @@
+            "HelloStarknet".to_string(),
+            Path::new("tests/data/contracts/hello_starknet.cairo"),
+        )
+        .unwrap()
+    );
+
+    let result = run_test_case(&test);
+
+    assert_passed!(result);
+}
+
+#[test]
+fn store_load_wrong_data_length() {
+    let test = test_utils::test_case!(
+        indoc!(
+            r#"
+            use starknet::ContractAddress;
+            use snforge_std::{ declare, ContractClassTrait, store, load };
+            
+            #[starknet::interface]
+            trait IHelloStarknet<TContractState> {
+                fn get_balance(ref self: TContractState) -> felt252;
+                fn increase_balance(ref self: TContractState, amount: felt252);
+            }
+            
+            fn deploy_contract() -> IHelloStarknetDispatcher {
+                let contract = declare('HelloStarknet');
+                let contract_address = contract.deploy(@array![]).unwrap();
+                IHelloStarknetDispatcher { contract_address }
+            }
+
+            #[test]
+            fn store_load_wrong_data_length() {
+                let deployed = deploy_contract();
+                
+                let stored_balance = deployed.get_balance();
+                assert(stored_balance == 0, 'wrong balance stored'); // No change expected
+                deployed.increase_balance(420);
+                
+                let loaded = load(deployed.contract_address, selector!("balance"), 2);
+                 // Even though wrong length is called, the first felt will be correct and second one uninitialized
+                assert(*loaded.at(0) == 420, 'wrong storage value');
+                assert(*loaded.at(1) == 0, 'wrong storage value');
+            }
+        "#
+        ),
+        Contract::from_code_path(
+            "HelloStarknet".to_string(),
+            Path::new("tests/data/contracts/hello_starknet.cairo"),
+        )
+        .unwrap()
+    );
+
+    let result = run_test_case(&test);
+
+    assert_passed!(result);
+}
+
+#[test]
+fn store_load_max_boundaries_input() {
+    let test = test_utils::test_case!(
+        indoc!(
+            r"
+            use starknet::ContractAddress;
+            use snforge_std::{ declare, ContractClassTrait, store, load };
+            
+            #[starknet::interface]
+            trait IHelloStarknet<TContractState> {
+                fn get_balance(ref self: TContractState) -> felt252;
+                fn increase_balance(ref self: TContractState, amount: felt252);
+            }
+            
+            fn deploy_contract() -> IHelloStarknetDispatcher {
+                let contract = declare('HelloStarknet');
+                let contract_address = contract.deploy(@array![]).unwrap();
+                IHelloStarknetDispatcher { contract_address }
+            }
+            
+            const MAX_FELT: felt252    = 3618502788666131213697322783095070105623107215331596699973092056135872020480;
```

Comment:
Not needed I think

Reply:
[Remove redundant const](https://github.com/foundry-rs/starknet-foundry/pull/1487/commits/3c22c9d7314bac722229e377e11e7616901a822d)